src/v2/Apps/Conversation/Components/ReviewOfferCTA.tsx

```diff
@@ -132,54 +118,47 @@ export const ReviewOfferCTA: React.FC<ReviewOfferCTAProps> = ({
     }
   }
 
-  const {
-    message,
-    subMessage,
-    backgroundColor,
-    Icon,
-    url,
-    modalTitle,
-  } = ctaAttributes
+  const { message, subMessage, backgroundColor, Icon } = ctaAttributes
 
-  const navigateToConversation = (path: string, title: string) => {
+  const openModal = () => {
     trackEvent(
       tappedViewOffer({
         impulse_conversation_id: conversationID,
         cta: message,
       })
     )
-
-    const { router } = useRouter()
-    router.push(`${path}?orderID=foo&title=bar`)
+    openOrderModal()
   }
 
   return (
-    <ClickableFlex
-      px={2}
-      py={1}
-      justifyContent="space-between"
-      alignItems="center"
-      bg={backgroundColor}
-      flexDirection="row"
-      minHeight={60}
-      onClick={() => {
-        navigateToConversation(url, modalTitle)
-      }}
-    >
-      <Flex flexDirection="row">
-        <Icon mt="3px" fill="white100" />
-        <Flex flexDirection="column" pl={1}>
-          <Text color="white100" variant="mediumText">
-            {message}
-          </Text>
-          <Text color="white100" variant="caption">
-            {subMessage}
-          </Text>
+    <>
+      <ClickableFlex
+        px={2}
+        py={1}
+        justifyContent="space-between"
+        alignItems="center"
+        bg={backgroundColor}
+        flexDirection="row"
+        minHeight={60}
+        onClick={() => {
```

Comment:
You mentioned about test in your PR description. Maybe you could simulate this click the check if `<StyledIframe>`  is present?